README.md

```diff
@@ -53,7 +57,53 @@ The plugin creates new tasks that you can use:
 
 For advanced usage please take a look into the sample project [build.gradle](sample/app/build.gradle) file.
 
-## Links
+Configuration
+-------------
+
+### Input Scripting
+
+The plugin has a extension called `script` which allows you to do simple scripting automation in a connected device.
+Here is an example called `autoLogin` which will input the test username and password into the sample app.
+
+```groovy
+inputScripts {
+    autoLogin {
+        inputScript {
```

Comment:
Here is how the new `NamedDomainObjectContainer` is used. Unfortunately I had to create another `inputScript` property in the middle to make the execution of the Closure lazy.

When I remove it and have the closure in one level up (like below), `configure` method of `NamedDomainObjectContainer` just executes the whole closure.

```groovy

        inputScripts {
            autoLogin {
                2.times {
                    text 'bob'
                    enter()
                }
                enter()
            }
        }
```